### PR TITLE
REVIEW: XMPP Newsletter April 2023

### PR DESCRIPTION
**Pull request by Echolon**
# XMPP Newsletter

## This is the draft for the next release covering April 2023

*Final release is planned for 05 05 2023.*

Contributions are much appreciated. You don't need any technical knowledge to contribute. If you need help, please ask in the Communication-Team group chat: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org).

If you would like to add news, that's possible via:

- Using our [online Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung)
- Adding your content to this pull request's thread (i.e. a comment below)
- Creating a pull request
- Posting something in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

It's really helpful if you try to give a short summary (<5 lines) of your contribution or of what the link you're posting is offering. You don't need to write in a formal tone - just be passionate about why this motivates you or about what you are working on!

| April '23         | 1st - 2nd May '23     | 3rd - 5th May '23 | 5th May '23 |
|--------------|-----------|------------|------------|
| Create Draft in Pad & GitHub | Last Call for final edits     | Review phase        |RELEASE! :rocket:|
| Collect news & highlights      |  |  Translators can start working      | |
| Reminder mid-month + end of month      |  |        ||

## Status

- [x] Collection of news
- [x] Review
- [ ] Blog post + Newsletter mail (use `tools/newsletter_email.py` after the post has been published)
- [ ] Social media
- [ ] Translations

### CODE REVIEW

File: content/blog/2023-05-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+## XSF Announcements
+
+- xmpp.org got a new software section! Looking for XMPP software, i.e. clients, servers, libraries, components, and tools? Check out [xmpp.org](https://xmpp.org/)'s new [software section](https://xmpp.org/software/), which lets you filter software by your own criteria. Looking for a client which works on Android and supports audio/video calls? Looking for a library that supports [XEP-0461: Message Replies](https://xmpp.org/extensions/xep-0461.html)? Just apply the filter and see what you get!
+
+{{< figure src="/images/newsletter/2023-03-05/xmpp-software-section.png" caption="xmpp.org's new software section" >}}
+
+- If you are interested to become an [XSF member](https://xmpp.org/community/membership/) you can [apply during this quarter until May 21th, 2023, 00:00 UTC](https://wiki.xmpp.org/web/Membership_Applications_Q2_2023).
+
+### XMPP and Google Summer of Code 2023
+The XSF has been [accepted again as hosting organisation at the GSoC 2023](https://xmpp.org/2023/02/xmpp-at-google-summer-of-code-2023/) !
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:21]:
```suggestion
The XSF has been [accepted again as hosting organisation at the GSoC 2023](https://xmpp.org/2023/02/xmpp-at-google-summer-of-code-2023/)!
```

File: content/blog/2023-05-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+{{< figure src="/images/logos/gsoc-2023-logo.svg" caption="XSF and Google Summer of Code 2023" >}}
+
+### XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:27]:
```suggestion
The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. We're open for applications via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
```

File: content/blog/2023-05-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## XMPP Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+- [FOSSY](https://fossy.us) will have an XMPP track at their conference this summer. Please [submit talk proposals](https://2023.fossy.us/call-for-proposals/) by May 14th.  Some financial support will be available from the track organizers for presenters who need it.
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:36]:
```suggestion
- [FOSSY](https://fossy.us) will have an XMPP track at their conference this summer. Please [submit talk proposals](https://2023.fossy.us/call-for-proposals/) by May 14th. The track organizers offer financial support for presenters, if needed.
```

File: content/blog/2023-05-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+- [XMPP Italian happy hour](https://tube.nicfab.eu/w/3KJMxZPYxRXQDAhXySZgLf): monthly Italian XMPP web meeting, starting May 16th and then every third Tuesday of the month at 7:00 PM. Online event, with web meeting mode and live streaming.
+
+## XMPP Sprints
+
+- [Elbe-Sprint Hamburg 2023](https://xmpp.org/2023/04/elbe-sprint-hamburg-2023/): Thursday, 22-06-2023 18:00 CEST — Sunday, 25-06-2023 12:00 CEST
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:41]:
```suggestion
- [Elbe-Sprint Hamburg 2023](https://xmpp.org/2023/04/elbe-sprint-hamburg-2023/): Thursday, 22-06-2023 18:00 CEST — Sunday, 25-06-2023 12:00 CEST. This summer, XMPP developers are holding a development sprint in Hamburg, Germany.
```

File: content/blog/2023-05-05-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    - One [video [DE]](https://www.eversten.net/xmppvideo/) explains some main aspects of XMPP.
+    - A series of [four videos [DE]](https://www.eversten.net/blog/monalvideos/) explains how iOS users can start using XMPP by installing and configuring the messenger app Monal.
+
+## Articles
+
+- A draft of the EU chat-control [has been published for download at Netzpolitik.org.](https://cdn.netzpolitik.org/wp-upload/2023/04/2023-04-05_EPRS_CSAM_Complementary-Impact-Assessment_DRAFT.pdf). Those are comment by 
+  - [Netzpolitik.org](https://netzpolitik.org/2023/chatkontrolle-studie-zerlegt-plaene-der-eu-kommission/)[DE]
+  - [Patrick Beyer (politician)](https://www.patrick-breyer.de/wissenschaftlicher-dienst-des-eu-parlaments-bestaetigt-chatkontrolle-verstoesst-gegen-grundrechte/) [DE]
+- Axel Reimer introduced a new German website called [xmpp24.de [DE]](https://xmpp24.de/). This website focuses on helping new XMPP users who want to start using XMPP on their Android or iOS devices. It explains exactly one onboarding flow (as a video tutorial) for each operating system. See aboves section.
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:54]:
```suggestion
- Axel Reimer introduced a new German website called [xmpp24.de [DE]](https://xmpp24.de/). This website focuses on helping new XMPP users who want to start using XMPP on their Android or iOS devices. It explains one onboarding flow (as a video tutorial) per operating system.
```

File: content/blog/2023-05-05-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+- The [JMP April Newsletter](http://blog.jmp.chat/b/april-newsletter-2023) talks about several developments, including a new MMS stack in testing, integration with the [Quicksy](https://quicksy.im) directory, the ability to create [Snikket](https://snikket.org) instances from inside a Cheogram Android onboarding flow, and an experimental [WebXDC](https://webxdc.org) prototype.
+
+## Software news
+### Clients and applications
+
+- [Gajim 1.7.3](https://gajim.org/post/2023-04-04-gajim-1.7.3-released/) has been released.
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:60]:
```suggestion
- [Gajim 1.7.3](https://gajim.org/post/2023-04-04-gajim-1.7.3-released/) has been released. This release enables you to mute notifications for specific contacts and brings some improvements and bug fixes.
```

File: content/blog/2023-05-05-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+### Servers
+
+- [ejabberd 23.04](https://www.process-one.net/blog/ejabberd-23-04/) has been released. This a big new release with many changes including support XEP-0425 (Message moderation), Real-Time Block List for MUC rooms and several SQL improvements.
+
+### Libraries & Tools
+
+- [QXmpp 1.5.4](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.4) and [QXmpp 1.5.5](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.5) have been released! They fix some important bugs.
+
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:69]:
```suggestion
```

File: content/blog/2023-05-05-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+- [Web Consosole Chat](https://soft.trung.fun/webcc/webcc.en.html) has been released. This is an installation guide and a collection of patches to make sure existing XMPP console clients are safe enough to before serving them to the web with [`ttyd`](https://github.com/tsl0922/ttyd) for your web-chat service.
```
Comment (cal0pteryx) [inline content/blog/2023-05-05-newsletter.md:70]:
```suggestion
- [Web Console Chat](https://soft.trung.fun/webcc/webcc.en.html) has been released. This is an installation guide and a collection of patches to make sure existing XMPP console clients are safe enough before serving them to the web with [`ttyd`](https://github.com/tsl0922/ttyd) for your web-chat service.
```

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter April '23 release

Many many thanks to everyone for their help in creation, review and translation!!!

This is a community effort, and we would like to thank translators for their contributions. Volunteers are welcome! Translations of the XMPP Newsletter will be released here (with some delay):

- English (original): [xmpp.org](https://xmpp.org/categories/newsletter/)
  - General contributors: **Adrien Bourmault (neox), Alexander "PapaTutuWawa", Arne, emus, Licaon_Kter, Ludovic Bocquet, melvo, MSavoritias (fae,ve), wurstsalat, XSF iTeam**
- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
  - Translators: **Adrien Bourmault (neox), alkino, anubis, Benoît Sibaud, bobble bubble, Nÿco, Pierre Jarillon, Ppjet6, Ysabeau**  
- German: [xmpp.org](https://xmpp.org/categories/newsletter/) and [anoxinon.de](https://anoxinon.de/blog/)
  - Translators: **Jeybe, wh0nix**
- Italian: [notes.nicfab.eu](https://notes.nicfab.eu)
  - Translators: **nicfab**
- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)
  - Translators: **daimonduff, TheCoffeMaker**

- :heart: :fireworks:  Next release will be next year on **5th of June 2023**!

Have a good time until then and see you soon!

**Pull request merged**